core/block_dragger.js

```diff
@@ -307,6 +300,41 @@ Blockly.BlockDragger.prototype.endDrag = function(e, currentDragDeltaXY) {
   Blockly.Events.setGroup(false);
 };
 
+/**
+ * Calculates the drag delta and new location values after a block is dragged.
+ * If a block would not be moved because a drag target prevents block moves,
+ * then there is no delta, and the new location is the same as the startXY.
+ * @param {!Blockly.utils.Coordinate} currentDragDeltaXY How far the pointer has
+ *     moved from the start of the drag, in pixel units.
+ * @return {{delta: (!Blockly.utils.Coordinate|undefined), newLocation:
+ *     !Blockly.utils.Coordinate}} New location after drag. delta is in
+ *     workspace units, and is undefined if a move is prevented. newLocation is
+ *     the new coordinate where the block should end up.
+ * @protected
+ */
+Blockly.BlockDragger.prototype.getNewLocationAfterMove_ = function(
```

Comment:
Instead of "AfterMove" I think it should reference that this is at the end of the drag, since there may not have been a  move and this method is called before the block is moved.

Reply:
Done, but let me know if you think of a better name..